src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -167,6 +170,10 @@ def __runner():
                             RobotStatus, robot_status
                         )
 
+                        _ = self.primitive_set_buffer.get(
```

Comment:
why don't we do anything with the primitiveset we get?

Reply:
The idea was to block until we get it I think

Reply:
please figure out what this is about and add a comment about it